File: Assets/Scripts/Buttons.js
#pragma strict
var Highscore : UnityEngine.UI.Text;

function Update () {
	Screen.sleepTimeout = SleepTimeout.NeverSleep;
	Highscore.text = "H i g h s c o r e : " + PlayerPrefs.GetInt("Best").ToString();
}

function QuitGame ()  {
	Debug.Log ("Game is exiting...");
	Application.Quit ();
}

function MainMenu() {
	Application.LoadLevel("MainMenu");
	Time.timeScale = 1;
}

function PLAY() {
	Application.LoadLevel("Endless");
}

File: Assets/Scripts/Buttons.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class Buttons : MonoBehaviour
{
    public UnityEngine.UI.Text Highscore;
    public virtual void Update()
    {
        Screen.sleepTimeout = SleepTimeout.NeverSleep;
        this.Highscore.text = "H i g h s c o r e : " + PlayerPrefs.GetInt("Best").ToString();
    }

    public virtual void QuitGame()
    {
        Debug.Log("Game is exiting...");
        Application.Quit();
    }

    public virtual void MainMenu()
    {
        Application.LoadLevel("MainMenu");
        Time.timeScale = 1;
    }

    public virtual void PLAY()
    {
        Application.LoadLevel("Endless");
    }

}